ci/backport.nix

```diff
@@ -0,0 +1,40 @@
+{ buildNpmPackage
+, fetchFromGitHub
+, lib
+, pkgs
+}:
+
+buildNpmPackage rec {
+  pname = "backport";
+  version = "9.6.6";
+
+  src = fetchFromGitHub {
+    owner = "sorenlouv";
+    repo = "backport";
+    rev = "v${version}";
+    hash = "sha256-VgEOUqbsgZ0EP9dN9iRmh+V05gEUaNhKASivt0pUKIw=";
+  };
+
+  dontNpmBuild = true;
+
+  # the compiled typescript files don't come in the release tags and neither does a package-lock.json
```

Comment:
If you have the commands on hand you used to produce the package-lock.json, it'd be good to drop those here so it's easy to update if need be.